enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,894 @@
+   2. then creating a service of service type load balancer with the annotation `service.beta.kubernetes.io/aws-load-balancer-eip-allocations`,
+      which uses the value from the field `eipAllocations` of `IngressController` CR.  
+
+#### 2. Update the EIP for an existing IngressController:
+   ##### Effectuating EIP Updates using manual method:
+   When the admin changes the eipAllocations field in the `IngressController` CR, the ingress operator sets the Load Balancer
+   Progressing status as per [section](#cluster-administrator-updates-eipallocations-field-of-the-existing-ingress-controller).
+   The cluster admin has to manually delete service in order to effectuate the change.
+   Then step 1.2 is followed by the ingress operator.
+   We have intentionally designed this feature to require manual deletion of the service for the following reasons:
+
+   Reason #1: To mitigate risks associated with cluster admins providing an invalid annotation value.
+   While the AWS Cloud Controller Manager (CCM) handles invalid annotations by producing errors and events, it is difficult 
+   for the Ingress Operator to decipher these CCM events after the load balancer has been provisioned. 
+   Therefore, the Ingress Operator can't indicate to a cluster admin that an IngressController's load balancer is in a 
+   malfunctioning state (i.e., the service is not getting reconciled) while the service is already provisioned.
+
+   When the IngressController is first created and the LoadBalancer-type service is not yet provisioned, 
+   these same invalid `eipAllocation` values will prevent the load balancer from being provisioned. 
+   The existing Ingress Operator logic will clearly indicate to the cluster admin that the load balancer failed to
+   provision via the LoadBalancerReady status condition. In addition, the LoadBalancerReady condition will include the CCM 
+   event logs that indicate to the cluster admin that the `eipAllocations` value is invalid (see Support Procedures for examples).
+   Therefore, requiring manual service deletion provides a way for the Ingress Operator to produce a clear signal to the
+   cluster admin that `eipAllocations` is invalid.
+
+   See [Invalid EIPAllocation Annotation Values](#invalid-eipallocation-values) for examples of invalid `eipAllocation` annotation values.
+
+   Reason #2: To mitigate upgrade compatibility issues.
+   Note: Directly configuring the `service.beta.kubernetes.io/aws-load-balancer-eip-allocations` on the LoadBalancer-type service is not
+   supported and never has been. However, this enhancement is designed to prevent cluster disruption upon upgrading with this
+   unsupported configuration.
+
+   If the `service.beta.kubernetes.io/aws-load-balancer-eip-allocations` annotation was previously configured and the cluster is upgraded,
+   the default value [] for EIPAllocations will clear the annotation for existing IngressControllers, which would break cluster ingress.
```

Comment:
Is this what you mean here?
```suggestion
   the empty default value for EIPAllocations will clear the annotation for existing IngressControllers, which would break cluster ingress.
```